spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -83,9 +82,13 @@ inline fun <reified T : Enum<T>> enumSetOf(vararg elems: T): EnumSet<T> =
  * Retrieve the full text for the license with the provided SPDX [id], including "LicenseRefs". If [handleExceptions] is
  * enabled, the [id] may also refer to an exception instead of a license.
  */
-fun getLicenseText(id: String, handleExceptions: Boolean = false) =
+fun getLicenseText(id: String, handleExceptions: Boolean = false): String? {
     if (id.startsWith("LicenseRef-")) {
-        object {}.javaClass.getResource("/licenserefs/$id")?.readText()
-    } else {
-        SpdxLicense.forId(id)?.text ?: SpdxLicenseException.forId(id)?.text?.takeIf { handleExceptions }
-    } ?: throw IOException("No license text found for id '$id'.")
+        return try {
```

Comment:
The `return` could even be moved to before the `if` (if you keep the `else`).

Reply:
done.